### PR TITLE
[Doc Improvement][Share to Teams from personal app or tab][3437956]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b94438ab7a7de51006ae4bc193f56b6420e7a6b9/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 89 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51411613-c07d-40e4-9ecc-013e4f1a4bc0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The following table provides the response codes:
 
 ## Limitations
 
-* Share to Teams is supported for desktop and web clients but isn't supported for mobile client.
+* Share to Teams is supported for desktop client but isn't supported for web and mobile clients.
```
Comment (v-deabira) [inline msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md:80]: The Share to Teams feature is available for the desktop client, but it's not supported on web and mobile clients.

**Review by v-deabira — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d271b61bd94901a21be3e3233dc4a3c7dd996b6e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 87 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a3c113e2-ae9a-444c-854d-067f5c0fc7a1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/70765d4341dff1616b75795b16d9d7df522fda6d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/106a861e-0419-4d0f-91cd-b83afab85f41) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca4208d8cd5ba2790bfeaaad797b6173a65de16/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 89 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/67f6ac4e-0641-495d-a403-c082a1909bf4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e14ccdc07844977602e298785d167326e21c001e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/17c013b0-bf31-4b5e-96c6-7de53603f31b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fc76e842f3ea8151cfffb1bd2cd3e6b0a28f4ed0/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-from-personal-app-or-tab.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7e71bff6-3c9a-4d71-a570-852e2ccbb4fb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**